src/components/Autocomplete.tsx

```diff
@@ -0,0 +1,110 @@
+import React, {
+  useCallback, useMemo, useRef, useState,
+} from 'react';
+import debounce from 'lodash.debounce';
```

Comment:
Fix it
<img width="866" alt="image" src="https://github.com/mate-academy/react_autocomplete/assets/64750958/661c5883-b543-4364-9fde-b00ca11f20c7">